interface/storage/pools-btrfs.rst

```diff
@@ -152,15 +230,15 @@ out more about each option from the `BTRFS documentation mount options section
 Pool Resize/ReRaid
 ------------------
 
-A convenience feature of btrfs Pool management is the ability to add or remove disks,
+A convenience feature of btrfs Pool management is the ability to add or remove drives,
 and change redundancy profiles, while still using the Pool.
 The persistence of a pool's accessibility is otherwise known as it's 'online' state.
 And so these changes are referenced as it's online capabilities.
 
 A performance reduction is expected during any changes of this sort,
 but depending on your hardware overhead, this can be unnoticeable.
 
-**Note that increases in; disk count, percent usage, snapshots count, and Pool size can all impact on the memory and CPU required,
+**Note that increases in; drive count, percent usage, snapshots count, and Pool size can all impact on the memory and CPU required,
```

Comment:
```suggestion
**Note that increases in drive count, percent usage, snapshots count, and Pool size can all impact on the memory and CPU required,
```

Reply:
I'm OK with leaving this as is, as the use of a semi-colon to introduce a list is legitimate and recognised in punctuation guides. Thought my use of the Oxford/Harvard/Serial comma is more likely to be questioned. But I like it :), as it helps to no conflate the last too list items - but I digress. Interestingly when I worked as a technical/grammatical/stylistic editor on some camera books, the exact same edit suggestion of my edits came up. They were also unaware of this use of the semi-colon. I had to dig out a grammar book in the middle of a meeting where all present, bar me, chose to ignore that recognised use. Oh well. Lets leave this one for now - at least until the ever developing trends in grammar rule out this possibly now archaic use of the semicolon.

Reply:
Absolutely! Let's leave it.